provisioning/roles/geerlingguy.drupal/tasks/build-composer-project.yml

```diff
@@ -1,4 +1,20 @@
 ---
+- name: Delete old /tmp/drupal-project
+  file:
+    path: "/tmp/drupal-project"
+    state: absent
+  when: not drupal_site_exists
```

Comment:
I run this before the other tasks so that it's always purged even if a followup task aborts the process.